src/main/java/net/javadiscord/javabot/systems/user_commands/format_code/FormatCodeCommand.java

```diff
@@ -49,7 +46,11 @@ public FormatCodeCommand() {
 								.addChoice("SQL", "sql")
 								.addChoice("Swift", "swift")
 								.addChoice("TypeScript", "typescript")
-								.addChoice("XML", "xml")
+								.addChoice("XML", "xml"),
+						new OptionData(OptionType.STRING,"auto-indent","The type of indentation that should be applied to the message, does not automatically indent if left blank",false)
```

Comment:
This fails at startup as field descriptions can only have 100 characters at maximum.